packages/ui5-application-writer/src/data/validators.ts

```diff
@@ -9,11 +11,27 @@ import { Ui5App } from '../types';
  */
 export function validateAppId(appId: string): boolean {
     if (!appId) {
-        throw new Error('The property: app.id must have a value'); // todo: add localised parameterised message
+        throw new Error(t('error.missingRequiredProperty', { propertyName: 'app.id' }));
```

Comment:
Test needed to cover : https://sonarcloud.io/code?id=SAP_open-ux-tools&pullRequest=319&selected=SAP_open-ux-tools%3Apackages%2Fui5-application-writer%2Fsrc%2Fdata%2Fvalidators.ts&line=33